build/open-vm-tools/patches/nicInfo.patch

```diff
@@ -31,7 +30,57 @@ diff -wpruN --no-dereference '--exclude=*.orig' a~/lib/nicInfo/nicInfoPosix.c a/
  static unsigned
  CountNetmaskBitsV6(struct sockaddr *netmask)
  {
-@@ -452,7 +449,7 @@ GuestInfoGetNicInfo(unsigned int maxIPv4
+@@ -306,7 +303,11 @@
+     * records are intermingled with AF_INET and AF_INET6 records.
+     */
+    for (pkt = ifaddrs; pkt != NULL; pkt = pkt->ifa_next) {
++#ifdef sun
```

Comment:
I'd prefer `__illumos__` or `__sun__` here, although I know our compilers define `sun` too.

Reply:
I was following what the patch for that file did already, can switch everything to `__illumos__` if needed.

Reply:
If there's already precedent for `sun`, let's leave it consistent. Thanks.